src/app/clusters/scenes/SceneTable.h

```diff
@@ -56,40 +56,52 @@ class SceneHandler
     SceneHandler(){};
     virtual ~SceneHandler() = default;
 
+    /// @brief Gets the list of supported clusters for an endpoint
+    /// @param endpoint target endpoint
+    /// @param clusterBuffer Buffer to hold the supported cluster IDs, cannot hold more than
+    /// CHIP_CONFIG_SCENES_MAX_CLUSTERS_PER_SCENES
+    virtual void GetSupportedClusters(EndpointId endpoint, Span<ClusterId> & clusterBuffer) = 0;

```

Comment:
Is the span resized smaller depending on the number of entries provided? Should mention that the ClusterIDs are copied.

Reply:
The functions is meant to adjust the span size using the reduce_size() method. This was added in the comment in latest commit.